### ISSUE TITLE
:sparkles: Add an MCDropout Baseline

### ISSUE THREAD

**Issue by o-laurent** [ENSTA-U2IS-AI/torch-uncertainty#40]
I have added dropout to standard ResNet in [de31452](https://github.com/ENSTA-U2IS/torch-uncertainty/commit/de314523bf9fc6ca6c06e4a4f3046097ac45446a). It could be interesting to add an MCDropout Ensemble baseline. To do this, the following steps could be considered:

- [ ] Create a new baseline in torch-uncertainty/torch_uncertainty/baselines/classification/resnet.py in cls.ensemble
- [ ] Find a solution to apply dropout in eval mode (potentially change nn.Dropout to F.dropout and replace self.training by true when the version is MCdropout)
- [ ] Add tests
- [ ] Add a tutorial to explain how to use the baseline
- [ ] Add the reference to the documentation
- [ ] Add a parameter to enable last-layer dropout
